code_samples/ai_actions/src/Command/ActionConfigurationCreateCommand.php

```diff
@@ -0,0 +1,93 @@
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ActionConfigurationCreateCommand extends Command
+{
+    protected static $defaultName = 'app:action-configuration-create';
+
+    private ActionConfigurationServiceInterface $actionConfigurationService;
+
+    private PermissionResolver $permissionResolver;
+
+    private UserService $userService;
+
+    private ActionServiceInterface $actionService;
+
+    private ActionTypeRegistryInterface $actionTypeRegistry;
+
+    public function __construct(
+        ActionConfigurationServiceInterface $actionConfigurationService,
+        PermissionResolver $permissionResolver,
+        UserService $userService,
+        ActionServiceInterface $actionService,
+        ActionTypeRegistryInterface $actionTypeRegistry
+    ) {
+        parent::__construct();
+        $this->actionConfigurationService = $actionConfigurationService;
+        $this->permissionResolver = $permissionResolver;
+        $this->userService = $userService;
+        $this->actionService = $actionService;
+        $this->actionTypeRegistry = $actionTypeRegistry;
+    }
+
+    protected function configure(): void
+    {
+        $this->addArgument('user', InputArgument::OPTIONAL, 'Login of the user executing the actions', 'admin');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $user = $input->getArgument('user');
+        $this->permissionResolver->setCurrentUserReference($this->userService->loadUserByLogin($user));
+
+        $refineTextActionType = $this->actionTypeRegistry->getActionType('refine_text');
+
+        $actionConfigurationCreateStruct = new ActionConfigurationCreateStruct('rewrite_casual');
+
+        $actionConfigurationCreateStruct->setType($refineTextActionType);
+        $actionConfigurationCreateStruct->setName('eng-GB', 'Rewrite in casual tone');
+        $actionConfigurationCreateStruct->setDescription('eng-GB', 'Rewrites the text using a casual tone');
+        $actionConfigurationCreateStruct->setActionHandler('openai-text-to-text');
+        $actionConfigurationCreateStruct->setActionHandlerOptions(new ArrayMap([
+            'max_tokens' => 4000,
+            'temperature' => 1,
+            'prompt' => 'Rewrite this content to improve readability. Preserve meaning and crucial information but use casual language accessible to a broader audience.',
+            'model' => 'gpt-4-turbo',
+        ]));
+        $actionConfigurationCreateStruct->setEnabled(true);
+
+        $this->actionConfigurationService->createActionConfiguration($actionConfigurationCreateStruct);
+
+        $action = new RefineTextAction(new Text([
+            <<<TEXT
+            Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, 
+            and which usually results in protein folding into a specific 3D structure that determines its activity.
+TEXT
```

Comment:
I would inject less white spaces.
```suggestion
<<<TEXT
Proteins differ from one another primarily in their sequence of amino acids, which is dictated by the nucleotide sequence of their genes, 
and which usually results in protein folding into a specific 3D structure that determines its activity.
TEXT
```